src/codeu/chat/client/commandline/Chat.java

```diff
@@ -425,7 +428,34 @@ public void invoke(List<String> args) {
         System.out.println("    Exit the program.");
       }
     });
-
+    panel.register("u-authorize", new Panel.Command()
+    {
+      @Override
+      public void invoke(List<String> args) 
+      {
+        if (args.size() < 2) 
+        {
+      	  System.out.println("ERROR: Please provide enough parameters!!!");
+      	} 
+        else
+        {
+          String user = args.get(0);
+          if(findUser(user) == null)
+          {
+            System.out.println("ERROR: No such user.");
+          }
+          else if(!args.get(1).equals("o") && !args.get(1).equals("u") && !args.get(1).equals("b"))
+          {
+            System.out.println(args.get(1));
+            System.out.println("ERROR: Parameter is unacceptable!");
```

Comment:
Once again, it would be more helpful if the string looked something like this:
"ERROR: Authority "+args.get(1)+" not supported. Use help to learn more."